filament/src/Material.cpp

```diff
@@ -378,6 +378,10 @@ Handle<HwProgram> FMaterial::getSurfaceProgramSlow(uint8_t variantKey) const noe
 
     if (Variant(variantKey).hasSkinningOrMorphing()) {
         pb.setUniformBlock(BindingPoints::PER_RENDERABLE_BONES, PerRenderableUibBone::_name);
+        pb.setUniformBlock(BindingPoints::PER_RENDERABLE_MORPHING, PerRenderableMorphingUib::_name);
```

Comment:
One thing I'm wondering here (maybe for a follow-up PR), we only need the MorphTargets texture in the vertex shader, so... it might not be too bad because in practice we're texture limited in the fragment shader.

From the GLES spec:
```
GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS
params returns one value, the maximum supported texture image units that can be used to access texture maps from the vertex shader and the fragment processor combined. If both the vertex shader and the fragment processing stage access the same texture image unit, then that counts as using two texture image units against this limit. The value must be at least 32.
```
So it  sounds like we can use 32 samplers/textures as long as it's 16 in each stage.

Now the problem is that filament doesn't distinguish between fragment and vertex shader. I need to think more about this.

Reply:
Right. I'm working on this at an other branch because the driver code needs to be chagned.

Reply:
It is solved by #5036.